cranelift-wasm/src/translation_utils.rs

```diff
@@ -76,6 +77,8 @@ pub enum GlobalInit {
     F32Const(u32),
     /// An `f64.const`.
     F64Const(u64),
+    /// A `vconst`.
+    V128Const(Uimm128),
```

Comment:
@sunfishcode, would you prefer this was `V128Const([u8; 16])` to avoid exposing our Uimm128 type?

Reply:
Ah. I commented about this in the Wasmtime PR. I think we can expose it, because it is just a raw value with no particular implementation details baked in. But we should rename it to `V128Imm` so that it's more descriptive of what  it represents.